vignettes/advs.Rmd

```diff
@@ -113,8 +123,17 @@ bmi_for_age <- who_bmi_for_age_boys %>%
         SEX == 2 ~ "F",
         TRUE ~ NA_character_
       ),
-      AGE = AGE * 30.4375
-    )) %>%
+      # Ensure first that Age unit is "DAYS"
+      AGE = round(AGE * 30.4375)
```

Comment:
AGEU must be added here (line117) instead of last step. as per updates in #73 , but also after line 118

Reply:
@Fanny-Gautier please can you make these vignette updates in your branch that is working on issue 73 specifically, as its hard to follow including that in this PR which is already doing so much. also that way i can review all those related changes in one PR together as i dont yet really understand why moving the AGEU bit is needed here. will be easier if i can review all together. i'm going to go ahead and merge this PR to main as all previous comments are resolved and then we can merge main into the other open PRs so all feature branches have all these latest changes available.